test/e2e/validation_test.go

```diff
@@ -53,6 +54,53 @@ type winService struct {
 	description string
 }
 
+// testNodesBecomeReady tests that all Windows nodes become ready and schedulable at least once.
+func (tc *testContext) testNodesBecomeReady(t *testing.T) {
+	nodes := gc.allNodes()
+	for index, node := range nodes {
+		t.Run(node.GetName(), func(t *testing.T) {
+			err := wait.PollImmediate(retry.Interval, retry.ResourceChangeTimeout, func() (done bool, err error) {
+				// Get current node state, and update it in the cache
+				foundNode, err := tc.client.K8s.CoreV1().Nodes().Get(context.TODO(), node.GetName(), meta.GetOptions{})
+				require.NoError(t, err)
+				nodes[index] = *foundNode
+				return tc.nodeReadyAndSchedulable(*foundNode), nil
```

Comment:
This implies, the function should be name `testNodesBecomeReadyAndSchedulable()`. Or am I missing something?